inbox/events/timezones.py

```diff
@@ -107,11 +110,160 @@
     'West Asia Standard Time': 'Asia/Tashkent',
     'West Pacific Standard Time': 'Pacific/Port_Moresby',
-    'Yakutsk Standard Time': 'Asia/Yakutsk',
+    'Yakutsk Standard Time': 'Asia/Yakutsk'
 }
 
+# Also add the friendly timezone names
+
+utc_friendly_timezones = {
+    '(UTC-12:00) International Date Line West': 'Etc/GMT+12',
+    '(UTC-11:00) Coordinated Universal Time-11': 'Etc/GMT+11',
+    '(UTC-10:00) Hawaii': 'Pacific/Honolulu',
+    '(UTC-09:00) Alaska': 'America/Anchorage',
+    '(UTC-08:00) Baja California': 'America/Los_Angeles',
+    '(UTC-08:00) Pacific Time (US & Canada)': 'America/Los_Angeles',
+    '(UTC-07:00) Arizona': 'America/Phoenix',
+    '(UTC-07:00) Chihuahua, La Paz, Mazatlan': 'America/Chihuahua',
+    '(UTC-07:00) Mountain Time (US & Canada)': 'America/Denver',
+    '(UTC-06:00) Central America': 'America/Guatemala',
+    '(UTC-06:00) Central Time (US & Canada)': 'America/Chicago',
+    '(UTC-06:00) Guadalajara, Mexico City, Monterrey': 'America/Mexico_City',
+    '(UTC-06:00) Saskatchewan': 'America/Regina',
+    '(UTC-05:00) Bogota, Lima, Quito': 'America/Bogota',
+    '(UTC-05:00) Bogota, Lima, Quito, Rio Branco': 'America/Bogota',
+    '(UTC-05:00) Eastern Time (US & Canada)': 'America/New_York',
+    '(UTC-05:00) Indiana (East)': 'America/Indianapolis',
+    '(UTC-04:30) Caracas': 'America/Caracas',
+    '(UTC-04:00) Asuncion': 'America/Asuncion',
+    '(UTC-04:00) Atlantic Time (Canada)': 'America/Halifax',
+    '(UTC-04:00) Cuiaba': 'America/Cuiaba',
+    '(UTC-04:00) Georgetown, La Paz, Manaus, San Juan': 'America/La_Paz',
+    '(UTC-03:00) Santiago': 'America/Santiago',
+    '(UTC-03:30) Newfoundland': 'America/St_Johns',
+    '(UTC-03:30) Newfoundland and Labrador': 'America/St_Johns',
+    '(UTC-03:00) Brasilia': 'America/Sao_Paulo',
+    '(UTC-03:00) Buenos Aires': 'America/Buenos_Aires',
+    '(UTC-03:00) Buenos Aires, Georgetown': 'America/Buenos_Aires',
+    '(UTC-03:00) Cayenne, Fortaleza': 'America/Cayenne',
+    '(UTC-03:00) Greenland': 'America/Godthab',
+    '(UTC-03:00) Montevideo': 'America/Montevideo',
+    '(UTC-03:00) Salvador': 'America/Bahia',
+    '(UTC-02:00) Coordinated Universal Time-02': 'America/Noronha',
+    '(UTC-02:00) Mid-Atlantic': 'Etc/GMT-2',
+    '(UTC-01:00) Azores': 'Atlantic/Azores',
+    '(UTC-01:00) Cabo Verde Is.': 'Atlantic/Cape_Verde',
+    '(UTC-01:00) Cape Verde Islands': 'Atlantic/Cape_Verde',
+    '(UTC) Casablanca': 'Africa/Casablanca',
+    '(UTC) Coordinated Universal Time': 'Etc/GMT',
+    '(UTC) Greenwich Mean Time: Dublin, Edinburgh, Lisbon, London': 'Etc/GMT',
+    '(UTC) Dublin, Edinburgh, Lisbon, London': 'Europe/London',
+    '(UTC) Monrovia, Reykjavik': 'Atlantic/Reykjavik',
+    '(UTC+01:00) Amsterdam, Berlin, Bern, Rome, Stockholm, Vienna': 'Europe/Berlin',
+    '(UTC+01:00) Belgrade, Bratislava, Budapest, Ljubljana, Prague': 'Europe/Budapest',
+    '(UTC+01:00) Brussels, Copenhagen, Madrid, Paris': 'Europe/Paris',
```

Comment:
Hmm yeah - technically this is casting CET to CST in the US, I think it'd be more accurate to set it to Etc/GMT+1 than to Europe/Paris which includes the DST. Will look at it tomorrow.